.github/workflows/e2e.yml

```diff
@@ -115,3 +115,4 @@ jobs:
           cd dev && tilt down || true
           docker volume prune -af || true
           docker rm -f $(docker ps -aq)
+          docker volume prune -af
```

Comment:
`docker system prune -a --volumes -f`

I think this is the best comment to remove everything